factcast-store/src/test/java/org/factcast/store/registry/transformation/chains/NOPRegistryMetrics.java

```diff
@@ -75,4 +76,9 @@ public void count(EVENT event, Tags tags) {}
 
   @Override
   public void increase(EVENT transformationCacheHit, int hits) {}
+
+  @Override
+  public ExecutorService monitor(ExecutorService executor, String name) {
```

Comment:
@nonnull ?

Reply:
I wanted to adhere to the surrounding code you know 👀😂